CHANGELOG.md

```diff
@@ -52,6 +52,8 @@ The format is based on [Keep a Changelog].
 * The user option `selectrum-complete-in-buffer` can be used to
   control whether Selectrum should handle in buffer completion (the
   default is t) ([#261]).
+* The user option `selectrum-default-value-format` can be used
+  to specify the formatting of the default value indicator.
```

Comment:
Please add the refs here and below in the link section.

Reply:
Sure, the PR did not exist when I wrote this.

Reply:
Yeah I know, but it's easily overlooked so I thought you might have forgotten it.